backend/codingchallenge/api/views/authentication.py

```diff
@@ -40,6 +41,11 @@ def post(self, request, *args, **kwargs):
             user = authenticate(request, username=username, password=password)
 
             if user:
+                if user.application.expiry > time.time():
+                    if user.application.status < Application.Status.ARCHIVED:
+                        user.application.status = Application.Status.EXPIRED
+                        user.save()
+                    return Response(jsonMessages.errorJsonResponse("The application is expired!"), status=status.HTTP_410_GONE)
```

Comment:
Here you use http code 410 but in the wiki it's 406

Reply:
I changed it. Thank you!